cmd/soroban-rpc/internal/methods/send_transaction.go

```diff
@@ -74,7 +74,11 @@ func NewSendTransactionHandler(daemon interfaces.Daemon, logger *log.Entry, stor
 		}
 		txHash := hex.EncodeToString(hash[:])
 
-		ledgerInfo := store.GetLatestLedger()
+		ledgerInfo, err := store.GetLatestLedger()
+		if err != nil {
+			return SendTransactionResponse{}, err
```

Comment:
just for thought, would be great if could add a test assertion for how this error is expected to look from json-rpc response, transaction_test.go has some other tests that check error responses, but in this case, not sure how to inject a mock or trigger GetLatestLedger to return an err during the test. 

Do you think it's worthwhile to wrap the err further into a jrpc2 struct, to include more Message denoting this was due to GetLatestLedger attempt similar to [invalid params above](https://github.com/stellar/soroban-tools/pull/1167/files#diff-73f019a9e55037d6a4812f6b0e82b5ed6e56ab3b8e0a0d6151406838b9cd2edeR61-R63)

Reply:
@sreuland Yes I was also thinking the same. It would be better for a user to see where it failed from.